Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN pushd /coreapi && \
     # needed for DB migrations
     find coreapi/ -mindepth 1 -maxdepth 1 \( ! -name 'alembic*' -a ! -name hack \) -exec rm -rf {} +
 
-RUN pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-worker.git@${F8A_WORKER_VERSION}
+RUN pip3 install git+https://github.com/sara-02/fabric8-analytics-worker.git@${F8A_WORKER_VERSION}
```

Comment:
let's not use forked repo branch here. Please create a PR in worker repo first. and Then use the short commit here